controls/V-72001.rb

```diff
@@ -63,20 +88,29 @@
 games:x:12:100:games:/usr/games:/sbin/nologin
 gopher:x:13:30:gopher:/var/gopher:/sbin/nologin
 
-Accounts such as \"games\" and \"gopher\" are not authorized accounts as they do not
-support authorized system functions.
+Accounts such as \"games\" and \"gopher\" are not authorized accounts as they
+do not support authorized system functions.
```

Comment:
Neat theory but I once officially load tested a network by playing Nexuiz since it had a decent network traffic pattern. Only the local ISSO is to say what is authorized or not.